openstax/settings/dev.py

```diff
@@ -11,3 +11,13 @@
     from .local import *
 except ImportError:
     pass
+
+##################################
+#   OVERRIDE ACCOUNTS SETTINGS   #
+##################################
+
+# use default loging and logout urls,
+# Needed for selenium test.
+ACC_APP_LOGIN_URL = None
```

Comment:
These three setting will never need to be overridden, only the KEY and SECRET variables need be set to None in dev.py.